src/main/java/xyz/aadev/generitech/common/config/ConfigWorldGen.java

```diff
@@ -50,54 +49,34 @@
 import java.util.Map;
 
 public final class ConfigWorldGen {
-    public static final Map<EnumOres, OreConfig> OreGenConfig = new HashMap<>(EnumOres.values().length);
+    protected static final Map<EnumOres, OreConfig> OreGenConfig = new HashMap<>(EnumOres.values().length);
     private static final Map<EnumOres, OreConfig> OreGenConfigDefaults = new HashMap<>(EnumOres.values().length);
     private static final int[] DEFAULT_DIMENSION_BLACKLIST = {-1, 1};
 
+    public static Map<EnumOres, OreConfig> getOreGenConfig() {
+        return OreGenConfig;
+    }
+
     public static ConfigCategory Worldgen;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make Worldgen a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "Worldgen" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1213)